src/internal/operations/common.go

```diff
@@ -0,0 +1,40 @@
+package operations
```

Comment:
This file seems extraneous since the func is only getting called in one place.

Reply:
it will be used by BackupOp in a few PRs